poc/tools/plot_prio3_multiproof_robustness.py

```diff
@@ -6,9 +6,9 @@
 
 import matplotlib.pyplot as plt
 
-from field import Field64, Field128
-from flp_bbcggi19 import FlpBBCGGI19
-from vdaf_prio3 import Prio3SumVec
+from vdaf_poc.field import Field64, Field128
+from vdaf_poc.flp_bbcggi19 import FlpBBCGGI19
+from vdaf_poc.vdaf_prio3 import Prio3SumVec
```

Comment:
From the `poc` directory, I tried running `sage -python tools/plot_prio3_multiproof_robustness.py`, and it didn't work. Running `sage -pip install -e .` first fixed it, though. I tried uninstalling the editable package, then copying the script up a directory, into `poc`, and running that worked as well. I suppose getting rid of the tools directory would be the simplest solution.

Reply:
Moved the plotter to the poc directory. There you can run it without installing the package.